### PR TITLE
Update the help text for the --node-osdisk-size parameter

### PR DESCRIPTION
**Pull request by theheatDK**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/command_modules/azure-cli-acs/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.1.2
+++++++
+* Updated the help text for the --node-osdisk-size parameter.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-acs/HISTORY.rst]: From the PR checklist, do not add history entries for help text changes. This should just say "Minor fixes."

Comment (theheatDK) [reply src/command_modules/azure-cli-acs/HISTORY.rst]: Thanks. I have tried to change it but Travis is not happy. I have no idea what I have done? :-)

**Comment by tjprescott** [comment]
Hmm… this was a help only PR but it looks like something funky happened to the commit history. It now includes commits for unrelated PRs. @troydai can you take a look and make sure it is safe to merge?  The meat of the PR was a help text change that is fine to merge.

**Comment by williexu** [comment]
@theheatDK Please make the PR against our `dev` branch, and NOT master

**Comment by theheatDK** [comment]
Sorry. I am new at this :-|
I will close this and create a new agains DEV.

**Pull request closed without merging**